core/src/main/scala/chisel3/Aggregate.scala

```diff
@@ -951,6 +951,14 @@ trait VecLike[T <: Data] extends IndexedSeq[T] with HasId with SourceInfoDoc {
   */
 abstract class Record extends Aggregate {
 
+  /** The list of parameter accessors used in the constructor of this [[chisel3.Record]].
+    * @note This is automatically overridden via the compiler plugin for user-defined bundles that mix-in [[chisel3.experimental.HasAutoTypename]],
+    *       and is meant for internal Chisel use only. Can not be manually overridden by users, or else an error will be thrown.
+    * @note This lives in Record rather than the [[chisel3.experimental.HasAutoTypename]] trait, due to compiler implementation details
+    *       preventing us from overriding a definition within a trait via the compiler plugin
+    */
```

Comment:
```suggestion
  /** The list of parameter accessors used in the constructor of this [[chisel3.Record]].
    *
    * @note This is automatically overridden via the compiler plugin for user-defined bundles that mix-in [[chisel3.experimental.HasAutoTypename]],
    *       and is meant for internal Chisel use only. Can not be manually overridden by users, or else an error will be thrown.
    * @note This lives in Record rather than the [[chisel3.experimental.HasAutoTypename]] trait, due to compiler implementation details
    *       preventing us from overriding a definition within a trait via the compiler plugin
    */
```

mega-nit to add a whitespace after first scaladoc line